src/extension/engine/mysql-pass.ts

```diff
@@ -115,6 +115,40 @@ export class MySQLType extends AbstractServer
         });
     }
 
+    /**
+     * @param {string} sql - queries
+     * @return {string[]}
+     */
+    splitQueries(sqlMulti: string) {
```

Comment:
For MySQL we can use flag `multipleStatements` in `createConnection` function. It seems to me that this is a safer solution.

https://www.npmjs.com/package/mysql#multiple-statement-queries

Reply:
I didn't know about that flag, it could ease the query operation but we would have to deal with `DELIMITER` command since [it looks not to be supported by the mysql module](https://github.com/mysqljs/mysql/issues/1683).
Moreover I don't know how much simple it would be, with the `multipleStatements` flag set, to link each element of the results array returned by the `.query()` call to each executed query (which can be useful for displaying the output to the user) without actually splitting the SQL string.
@Bajdzis what do you think about it?